app.py

```diff
@@ -67,17 +71,14 @@ def run(args):
 
             plugin.publish(TOPIC_CAR, cars, timestamp=timestamp)
             plugin.publish(TOPIC_PEDESTRIAN, pedestrians, timestamp=timestamp)
-            if args.debug:
-                print(f"Cars: {cars}, pedestrians: {pedestrians}")
-                print(f"Measures published")
+            logging.info("publish cars=%d, pedestrians=%d", cars, pedestrians)
```

Comment:
I would put this as a debug information as we expect this inferencing goes fast like 10 up to 30 fps. not sure if we want to print this that fast.. Well.. Let's keep it as is. It wouldn't hurt the system....

Reply:
Ok. We can just switch them to `logging.debug` instead of `logging.info`, if you think that's helpful.

Reply:
Another options is to occasionally print out some aggregate stats, too. Like every few seconds print the total so you know something's happening.